domains/cinego.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "email": "virajjadhav512@gmail.com",
+    "github": "DeveloperViraj"
```

this should be your github username
```suggestion
    "username": "DeveloperViraj"
```